lib/backend/model/bgppeer.go

```diff
@@ -33,111 +33,65 @@ var (
 	typeBGPPeer        = reflect.TypeOf(BGPPeer{})
 )
 
-type BGPPeerKey struct {
-	Scope    scope.Scope `json:"-" validate:"omitempty"`
-	Hostname string      `json:"-" validate:"omitempty"`
-	PeerIP   net.IP      `json:"-" validate:"required"`
+type HostBGPPeerKey struct {
```

Comment:
We can - I kept it as Host to be inline with other ones - but it was kinda bugging me.